src/types/ChargingStationWorker.ts

```diff
@@ -12,22 +13,40 @@ export interface ChargingStationWorkerData extends WorkerData {
   chargingStationWorkerOptions?: ChargingStationWorkerOptions;
 }
 
-enum InternalChargingStationWorkerMessageEvents {
+export interface ChargingStationSubData {
+  id: string;
+  stationInfo: ChargingStationInfo;
+}
+
+export interface ChargingStationData {
```

Comment:
Why you need that type apart and exported? 
The full payload should be used as input/output on message handlers.

Reply:
I use `ChargingStationSubData` to define the Map in `AbstractUIServer`.
`chargingStations: Map<string, ChargingStationSubData>;`